webpack/components/extensions/Hosts/BulkActions/BulkPackagesWizard/BulkPackagesWizard.js

```diff
@@ -62,18 +63,18 @@ export const useHostsBulkSelect = ({ initialSelectedHosts, modalIsOpen }) => {
   };
 };
 
-export const getPackagesUrl = (selectedAction) => {
+export const getPackagesUrl = (selectedAction, osFamily) => {
+  if (osFamily === BULK_ACTIONS_OSFAMILY.INVALID) { return ''; }
```

Comment:
```suggestion
  if (osFamily === BULK_ACTIONS_OSFAMILY.INVALID) return '';
```

Curly braces are not required for single-line if statements. (I thought our eslint rules would catch this, hmm.)